src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -312,9 +304,32 @@ export const TradeInput = memo(() => {
           (!applyThorSwapAffiliateFees || activeSwapperName !== SwapperName.Thorchain) && (
             <DonationCheckbox isLoading={isLoading} />
           )}
+        {!!sortedQuotes.length && hasUserEnteredAmount && (
+          <Accordion allowToggle defaultIndex={0}>
+            <AccordionItem>
+              <h2>
+                <Center>
+                  <AccordionButton>
+                    <Box as='span' flex='1' textAlign='left'>
+                      <Text
+                        translation={'trade.availableRoutes'}
+                        fontWeight={'semibold'}
+                        color='text.subtle'
+                        _hover={{ color: hoverColor }}
```

Comment:
this needs a useMemo

Reply:
https://github.com/shapeshift/web/pull/5163/commits/54ebb1c6bc2cb861bccabf96e8e3906f49439c0d